config/config.yml.sample

```diff
@@ -1,5 +1,14 @@
 defaults: &defaults
-
+  account_colors:
+    negative:
+      css_class: 'negative'
```

Comment:
@Gataniel I think we don't need this class variable here... Just store color value.